disabled/disabledPublisher.go

```diff
@@ -0,0 +1,30 @@
+package disabled
+
+import "github.com/ElrondNetwork/notifier-go/data"
+
+// Publisher defines a disabled publisher component
+type Publisher struct{}
+
+// Run does nothing
+func (dp *Publisher) Run() {
+}
+
+// BroadcastChan returns a nil channel
+func (dp *Publisher) BroadcastChan() chan<- data.BlockEvents {
+	return nil
```

Comment:
Also here: is it safe to return a nil chan?

Reply:
modified based on suggestion :+1: